test/uglify/cache.test.js

```diff
@@ -1,30 +0,0 @@
-import crypto from 'crypto';
```

Comment:
@michael-ciniawsky be good add more tests, but `cache.test.js` doesn't test caching (just test `get` and `put` function)